docs/development/development-environment-docker/README.md

```diff
@@ -385,6 +392,16 @@ After amending the override file and the `.env`, ensure that you restart the sta
 docker compose up -d frontend
 ```
 
+### Adding a new service
+
+Some development tasks require you to run separate services that interact with OpenProject. For example, you might want
+to have Nextcloud running to test the Nextcloud-OpenProject integration. To do this, you'll need to follow some steps:
+
+1. Add the Nextcloud service to your `docker-compose.override.yml`, with the appropriate traefik labels, network, and
+   ca-bundle mounted.
```

Comment:
I'd challenge the idea of putting additional services into your core override. I'd at least show the second option, too: adding new services as seperate stacks (different docker compose, same external network)

Reply:
Feel free to do this, but I think that using the docs to explain the simplest possible setup is best, and then people can still fumble it all into different stacks with separate networks yada yada if they like